Assets/Tests/EditMode/Lanelet/LaneletTest.cs

```diff
@@ -0,0 +1,212 @@
+using NUnit.Framework;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor.SceneManagement;
+using UnityEngine.SceneManagement;
+using UnityEngine;
+using UnityEngine.TestTools;
+using UnityEngine.TestTools.Utils;
+using AWSIM.TrafficSimulation;
+using AWSIM;
+using AWSIM.Lanelet;
+using UnityEditor;
+
+public class LaneletTest
```

Comment:
@szylis 
Could you add a comment explaining the content of this test?

Reply:
@mackierx111 Done